src/test/java/org/sonar/plugins/clojure/language/ClojureSonarWayProfileTest.java

```diff
@@ -54,4 +54,14 @@ public void testIfSonarwayProfileIsCreatedWithAllEastwoodRules() {
 
         ruleKeys.stream().forEach(eastwoodRule -> assertTrue(ruleKeys.contains(eastwoodRule)));
     }
+
+    @Test
+    public void testIfSonarwayProfileIsCreatedWithAllAncientCljRules() {
```

Comment:
I think this test is not really testing anything. I just saw that `testIfSonarwayProfileIsCreatedWithAllEastwoodRules` has the same problem. lol

Reply:
Both tests have been fixed.